tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -727,10 +727,25 @@ test.describe( 'Nested Tabs tests', () => {
 		await page.waitForSelector( '.elementor-widget-n-tabs' );
 		// Test on desktop.
 		await expect( page.locator( '.e-normal.e-active' ) ).toHaveClass( 'e-n-tab-title e-normal elementor-animation-grow e-active' );
+
+		// Test the hover animation.
+		await page.locator( '.e-normal:not( .e-active )' ).last().hover();
+		const tabHover = await page.locator( '.e-normal:not( .e-active )' ).last().evaluate( ( element ) => {
```

Comment:
You have duplicated code in the two lines above, you can use set a const with the last not active tab and use.
Something like:

```
const lastNotActiveTab = page.locator( '.e-normal:not( .e-active )' ).last();
		await lastNotActiveTab.hover();
		const tabHover = await lastNotActiveTab.evaluate( ( element ) => {
			const animationValue = window.getComputedStyle( element ).getPropertyValue( 'transform' );
			
```
You can add it to the test`Arrange` section.

Reply:
Updated

Reply:
Can't you use `const tabHover = await tabNormal.evaluate( ( element ) => {`?